app/main/views/suppliers.py

```diff
@@ -23,10 +23,28 @@ def list_suppliers():
 
     prefix = request.args.get('prefix', '')
 
-    suppliers = Supplier.query.order_by(Supplier.name)
+    framework = request.args.get('framework')
+
+    if framework:
+        is_valid_string_or_400(framework)
+
+        active_services = Service.query.join(
+            Service.framework
+        ).filter(
+            Framework.status == 'live',
```

Comment:
If we're querying by framework why restrict it only to 'live' frameworks at the API?